components/bl-granim-component/README.md

```diff
@@ -0,0 +1,76 @@
+# Granim
+
+Granim is the component that can be used in Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). It allows you to create fluid and interactive gradient animations in your application. You will be able to apply your own animation settings to any of the blocks of your own application or leave as default. In any case you will get more colors and interactant on the pages of your application.
+More information about granim you can find in the [Granim](https://github.com/sarcadass/granim.js) library.
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
+</p>
+
+## Properties
+
+| Property                                                 | Type                                                                                                                                                                                                                      | Default value | Logic                  | Data Binding | UI Setting | Description                                                                                                                                                                                                                                                                          |
+|----------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|------------------------|--------------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| States <br> `states`                                     | *JSON*                                                                                                                                                                                                                    | `{}`          | States Logic           | YES          | NO         | This is a handler for the logic to determine the object containing all the states. Signature of states: {}. Signature of state: { gradients: [...[...String]] \|\| [...{color: String, pos: Number}], transitionSpeed: Number, loop: Boolean}. Watch [Codeless Examples](#Examples). |
```

Comment:
Looks like mistake: `Signature of states: {}`

Also we forgot for code formatting for signatures.